controllers/articles-controller.js

```diff
@@ -4,10 +4,12 @@ const getArticle = (request, response, next) => {
     const {article_id} = request.params;
 
     selectArticle(article_id).then((article) => {
-        const singleArticle = {article}
-         response.status(200).send(singleArticle); 
+         response.status(200).send({article}); 
         })
         .catch((err) => {
+            if (err.code === '22P02') {
+              err = {status: 400}  
+            }
             next(err);
```

Comment:
This logic could go into your main `app.js` error handler middleware, rather than specifically in this 1 controller.

Our notes on error handling help to visualise this: https://notes.northcoders.com/courses/js-back-end/error-handling

Reply:
It's good that you've got `next(err)` invoked, but the logic above it could sit in your main error handler middleware